kubeflow/pytorch-job/pytorch-operator.libsonnet

```diff
@@ -0,0 +1,281 @@
+{
+  all(params):: [
```

Comment:
Pass in env as well and get the namespace from env. We should no longer need to use params for the namespace